language/portuguese/form_validation_lang.php

```diff
@@ -10,31 +10,32 @@
 defined('BASEPATH') OR exit('No direct script access allowed');
 
 $lang['form_validation_required']               = 'O campo {field} é obrigatório.';
-$lang['form_validation_isset']                  = 'O campo {field} tem que conter um valor.';
-$lang['form_validation_valid_email']            = 'O campo {field} tem que conter um email válido.';
-$lang['form_validation_valid_emails']           = 'O campo {field} tem que conter todos os emails válidos.';
-$lang['form_validation_valid_url']              = 'O campo {field} tem que conter um URL válido.';
-$lang['form_validation_valid_ip']               = 'O campo {field} tem que conter um IP válido.';
-$lang['form_validation_min_length']             = 'O campo {field} tem que conter pelo menos {param} caractéres de comprimento.';
-$lang['form_validation_max_length']             = 'O campo {field} não pode ter mais que {param} caractéres de comprimento.';
-$lang['form_validation_exact_length']           = 'O campo {field} te que ter exatamente {param} caractéres de comprimento.';
-$lang['form_validation_alpha']                  = 'O campo {field} só pode conter caractéres do alfabeto.';
-$lang['form_validation_alpha_numeric']          = 'O campo {field} só pode conter caractéres alfa-numéricos.';
-$lang['form_validation_alpha_numeric_spaces']   = 'O campo {field} só pode conter caractéres alfa-numéricos e espaços.';
-$lang['form_validation_alpha_dash']             = 'O campo {field} só pode conter caractéres alfa-numéricos, underscores e hifens.';
+$lang['form_validation_isset']                  = 'O campo {field} deve conter um valor.';
+$lang['form_validation_valid_email']            = 'O campo {field} deve conter um email válido.';
+$lang['form_validation_valid_emails']           = 'O campo {field} deve conter todos os emails válidos.';
+$lang['form_validation_valid_url']              = 'O campo {field} deve conter um URL válido.';
+$lang['form_validation_valid_ip']               = 'O campo {field} deve conter um IP válido.';
+$lang['form_validation_valid_mac']              = 'O campo {field} deve conter um MAC válido.';
+$lang['form_validation_min_length']             = 'O campo {field} deve conter pelo menos {param} caracteres.';
```

Comment:
there's a typo in the word caracteres: you should write "caractéres".

Reply:
That is not correct. Not in pt-pt at least